src/redist/targets/Branding.targets

```
@@ -7,6 +7,7 @@
       <SharedFrameworkBrandName>Microsoft .NET Core Runtime $(MicrosoftNETCoreAppPackageVersion)</SharedFrameworkBrandName>
       <NetCoreAppTargetingPackBrandName>Microsoft .NET Core Targeting Pack $(MicrosoftNETCoreAppPackageVersion)</NetCoreAppTargetingPackBrandName>
       <NetStandardTargetingPackBrandName>Microsoft .NET Standard 2.1 Targeting Pack $(MicrosoftNETCoreAppPackageVersion)</NetStandardTargetingPackBrandName>
+      <NetCoreAppHostPackBrandName>Microsoft .NET App Host Pack $(MicrosoftNETCoreAppPackageVersion)</NetCoreAppHostPackBrandName>
```

```suggestion
      <NetCoreAppHostPackBrandName>Microsoft .NET AppHost Pack $(MicrosoftNETCoreAppPackageVersion)</NetCoreAppHostPackBrandName>
```

At least, it's one word [in Core-Setup](https://github.com/dotnet/core-setup/blob/29db80e003d7160f20bd2b036ba16721c3ee66bc/src/pkg/dir.props#L93), and https://github.com/dotnet/designs/pull/50 refers to them that way too.